moderngl_window/timers/clock.py

```diff
@@ -72,7 +72,13 @@ def next_frame(self) -> tuple[float, float]:
         self._frames += 1
         current = self.time
         delta, self._last_frame = current - self._last_frame, current
-        self._fps = 1.0 / delta
+
+        # Avoid division by zero on first frame
+        if delta > 0:
+            self._fps = 1.0 / delta
+        else:
+            self._fps = 0.0
+

```

Comment:
`timers/clock.py` is cover by utest `tests/test_timers.py` maybe you can update this utest to integrate your fix on `delta` (just one example (`delta = 0`) who used this patch)

Reply:
I approve of anyone making tests to cover my sloppiness.

Reply:
done ✅